test/msw-api/setup-server/life-cycle-events/on.test.ts

```diff
@@ -50,6 +53,12 @@ beforeAll(async () => {
     listener(`[request:unhandled] ${req.method} ${req.url.href} ${req.id}`)
   })
 
+  server.events.on('request:unhandledException', (req, handler, error) => {
```

Comment:
I don't like the way I've modeled these tests in the past. Instead of invoking a listener in this callback function's body, I should've passed the listener as the callback itself. This would allow us to assert _all_ arguments passed along a particular event.

```js
server.events.on('foo', listener)

expect(listener).toHaveBeenCalledWith([mockedRequest, handler, error])
```

Let's refactor this after this feature 👍